### DISCUSSION TITLE
[FEATURE REQUEST] Helper is missing a parameter that only AngleDraw has

### DISCUSSION THREAD

**Discussion by SLIFINITY** [ikemen-engine/Ikemen-GO#1984] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

For some reason, the Helper state controller does not have a RescaleClsn parameter but AngleDraw does, I would say it was a missed opportunity.

### Describe the solution you'd like

Since AngleDraw's RescaleClsn is fully added now in the source code, it should be a trivial effort to add that to the Helper state controller as well.
There may be some adjustments that could be made so that both AngleDraw and Helper's RescaleClsn are compatible with each other, but it should not be very hard to do.

### Additional context

I am working on a remake of a very old Minigame-like thing from way back in my early MUGEN years, which is Garfields, I made it for fun just to screw around and to be honest...I had to take some coding liberties to compensate for MUGEN's lack of clsn rescaling on helpers with different x and y scales.

**Comment by potsmugen** [comment]
I played with this idea back when I added RescaleClsn, but wasn't sure how to implement it on the user end. A more accurate parameter name would be something like `OwnClsnScale`. This would make the base clsn scale be the helper's own scale rather than the root's (it's a little more complicated but that mostly covers it).

**Comment by potsmugen** [comment, marked as answer]
Already implemented.
